docs/endpoints/multi_user.rst

```diff
@@ -1,219 +1,154 @@
-lowering the barrier to use of the resource.
-
-From an SDK user's perspective, the change to use a MEP is minimal, requiring still the
-endpoint agent identifier, and optionally any template-variable values.  An example
-SDK interaction might look like:
-
-.. code-block:: python
-   :emphasize-lines: 7, 9
-
-   from globus_compute_sdk import Executor
-
-   def hello_from_node(name: str):
-       import os
-       return f"Hello, {name}; I ran on {os.uname().nodename}"
-
-   mep_id = "id_as_shared_by_mep_admin"  # perhaps via email, or a web site
-   with Executor() as ex:
-       ex.endpoint_id = mep_id  # no change from a SEP interaction
-       f = ex.submit(hello_from_node, "Obi-Wan")
-       print(f.result())
-
-This interaction is no different than for use of a SEP (in fact, the Tutorial Endpoint
-is a MEP, and this interaction works there as well!), although does not customize the
-configuration.  For that, the only necessary addition is the ``user_endpoint_config``:
+  This chapter describes Multi-user Compute Endpoints (MEP) for site administrators.
+  For those not running MEPs with ``root`` privileges, please instead consult the
+  :ref:`endpoints_templating_configuration` section in the previous chapter.
 
-.. code-block:: python
-   :emphasize-lines: 10
-
-   from globus_compute_sdk import Executor
-
-   def hello_from_node(name: str):
-       import os
-       return f"Hello, {name}; I ran on {os.uname().nodename}"
+Multi-user Compute Endpoints (MEP) enable administrators to securely offer compute
+resources to users without mandating the typical shell access (i.e., ssh).  The basic
+thrust is that the administrator of a MEP creates a configuration template that is
+populated by user data and passed to a user endpoint process, or UEP.  When a user
+sends a task to the MEP, it will start a UEP on their behalf as a local POSIX user
+account mapped from their Globus Auth identity.  In this manner, administrators may
+preset endpoint configuration options (e.g., Torque, PBS, Slurm) and offer
+user‑configurable items (e.g., account id, problem‑specific number of blocks), while at
+the same time lowering the barrier to use of the resource.
 
-   mep_id = "id_as_shared_by_mep_admin"  # perhaps via email, or a web site
-   with Executor() as ex:
-       ex.endpoint_id = mep_id  # no change from a SEP interaction
-       ex.user_endpoint_config = {"ACCOUNT_ID": "OBIWAN123"}
-       f = ex.submit(hello_from_node, "Obi-Wan")
-       print(f.result())
+.. note::
 
-More on the specifics later.  The rest of this section is oriented toward cluster
-administrators, and assumes a passable understanding of installation and use of a
-single-user Compute endpoint.
+   The only difference between a :abbr:`CEA (single-user Compute endpoint agent)` and a
+   UEP is a semantic one for discussion to differentiate the two different starting
+   contexts.  The CEA is started manually by a human, while a UEP will always have a
+   parent MEP process.
```

Comment:
Proposal to simplify some of this terminology:
- The Globus Compute agent is an installable package that creates Compute Endpoints.
- The agent launches single-user Endpoints by default.
- If specified, the agent will launch a multi-user Endpoint, which manages multiple single-user Endpoints.

Reply:
I'm hesitant to use the term "single-user".  We've discussed (correctly, I think) that at some point, the only difference among what is now SEP vs UEP vs MEP will become documentation&nbsp;&mdash;&nbsp;as in administrator documentation and user documentation.  As referenced in the note ("Pardon “the mess” while we build the product[...]") this is a step in that direction.

Meanwhile, I think I've addressed the CEA awkwardness in the work I'll push shortly.